models/emergency_contact.py

```diff
@@ -32,7 +32,7 @@ def json(self):
             'id': self.id,
             'name':self.name,
             'description': self.description,
-            'contact_numbers': [number.json() for number in self.contact_numbers]
+            'contactNumbers': [number.json() for number in self.contact_numbers]
```

Comment:
This could impact the FE. I think they like receiving the json keys in camelCase though. I know they're still working on integrating a lot of the pages so it may not have any impact at all.

Reply:
All of the other json keys use camel case, so it would be more consistent. But if FE developers have already realized that contact_numbers is the exception, then it makes sense to just keep it as contact_numbers.

Reply:
Personally I prefer snake case. :-) For the json response perhaps camel case is preferred by the FE. I noticed in some other places we are doing some weird stuff with the json responses, and after speaking with @Chris-Boe I'm not to worried about breaking the FE, however we still need to take note of it so that people who work on the FE don't waste time debugging. 

It looks like they're dependent on `contact_numbers` in two files:

- codeforpdx/dwellingly-app/src/views/emergencyContacts.js
- codeforpdx/dwellingly-app/src/views/addEmergencyContact.js

Reply:
We can either revert this and ignore the inconsistency in style, or get an issue created and/or a PR submitted to update the FE.

Reply:
I'll revert back to the snake case. If we feel that consistency is really an issue, we can create another issue.

Reply:
OK. sounds good.